src/participant_network/cl/prysm/prysm_launcher.star

```diff
@@ -0,0 +1,291 @@
+	beacon_metrics_url = "{0}:{1}".format(beacon_service.ip_address, beacon_metrics_port.number)
+
+	validator_metrics_port = validator_service.ports[VALIDATOR_MONITORING_PORT_ID]
+	validator_metrics_url = "{0}:{1}".format(validator_service.ip_address, validator_metrics_port.number)
+
+	beacon_node_metrics_info = new_cl_node_metrics_info(beacon_node_service_id, METRICS_PATH, beacon_metrics_url)
+	validator_node_metrics_info = new_cl_node_metrics_info(validator_node_service_id, METRICS_PATH, validator_metrics_url)
+	nodes_metrics_info = [beacon_node_metrics_info, validator_node_metrics_info]
+
+
+	result = new_cl_client_context(
+		"prysm",
+		beacon_node_enr,
+		beacon_service.ip_address,
+		HTTP_PORT_NUM,
+		nodes_metrics_info,
+		beacon_node_service_id
+	)
+
+	return result
+
+
+def get_beacon_config(
+		genesis_data,
+		beacon_image,
+		bootnode_context,
+		el_client_context,
+		mev_boost_context,
+		log_level,
+		extra_params,
+	):
+
+	el_client_engine_rpc_url_str = "http://{0}:{1}".format(
+		el_client_context.ip_addr,
+		el_client_context.engine_rpc_port_num,
+	)
+
+	genesis_config_filepath = path_join(GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER, genesis_data.config_yml_rel_filepath)
+	genesis_ssz_filepath = path_join(GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER, genesis_data.genesis_ssz_rel_filepath)
+	jwt_secret_filepath = path_join(GENESIS_DATA_MOUNT_DIRPATH_ON_SERVICE_CONTAINER, genesis_data.jwt_secret_rel_filepath)
+
+
+	cmd_args = [
+		"--accept-terms-of-use=true", #it's mandatory in order to run the node
+		"--datadir=" + CONSENSUS_DATA_DIRPATH_ON_SERVICE_CONTAINER,
+		"--chain-config-file=" + genesis_config_filepath,
+		"--genesis-state=" + genesis_ssz_filepath,
+		"--http-web3provider=" + el_client_engine_rpc_url_str,
+		"--rpc-host=" + PRIVATE_IP_ADDRESS_PLACEHOLDER,
+		"--rpc-port={0}".format(RPC_PORT_NUM),
+		"--grpc-gateway-host=0.0.0.0",
+		"--grpc-gateway-port={0}".format(HTTP_PORT_NUM),
+		"--p2p-tcp-port={0}".format(DISCOVERY_TCP_PORT_NUM),
+		"--p2p-udp-port={0}".format(DISCOVERY_UDP_PORT_NUM),
+		"--min-sync-peers={0}".format(MIN_PEERS),
+		"--verbosity=" + log_level,
+		# Set per Pari's recommendation to reduce noise
+		"--subscribe-all-subnets=true",
+		"--jwt-secret={0}".format(jwt_secret_filepath),
+		# vvvvvvvvv METRICS CONFIG vvvvvvvvvvvvvvvvvvvvv
+		"--disable-monitoring=false",
+		"--monitoring-host=0.0.0.0",
+		"--monitoring-port={0}".format(BEACON_MONITORING_PORT_NUM)
+		# ^^^^^^^^^^^^^^^^^^^ METRICS CONFIG ^^^^^^^^^^^^^^^^^^^^^
+	]
+
+	if bootnode_context != None:
+		cmd_args.append("--bootstrap-node="+bootnode_context.enr)
+
+	if mev_boost_context != None:
+		cmd_args.append(("--http-mev-relay{0}".format(mev_boost_endpoint(mev_boost_context))))
+
+	if len(extra_params) > 0:
+		# we do the for loop as otherwise its a proto repeated array
+		cmd_args.extend([param for param in extra_params])
+
+	return struct(
+		image = beacon_image,
+		ports = BEACON_NODE_USED_PORTS,
+		cmd_args = cmd_args,
```

Comment:
it's a bit weird that for prism we set `entry_point_args = ["sh", "-c"],` but here we don't isn't it?

Reply:
I am just following - https://github.com/kurtosis-tech/eth2-merge-kurtosis-module/blob/fda6eec31c2c28db4630080960a6d59378997015/kurtosis-module/impl/participant_network/cl/prysm/prysm_launcher.go

seems like prysm and nimbus are broken on the main repo anyway. I have issues for those
https://github.com/kurtosis-tech/eth2-merge-kurtosis-module/issues/156